recipes/reduct-cpp/all/patches/1.12.0-0001-use-cci.patch

```diff
@@ -0,0 +1,39 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index af71f65..7e3a0e2 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -16,7 +16,20 @@ set(CMAKE_CXX_STANDARD 20)
+ set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/bin)
+ set(CMAKE_LIBRARY_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/lib)
+ 
+-include(cmake/InstallDependencies.cmake)
++find_package(fmt REQUIRED CONFIG)
```

Comment:
What situation! We need to patch it in order to not use Conan! 

https://github.com/reductstore/reduct-cpp/blob/main/cmake/InstallDependencies.cmake#L12